iOS/Flows/People/PeopleViewController+Search.swift

```diff
@@ -16,8 +16,12 @@ extension PeopleViewController: UISearchBarDelegate {
     
     func searchBarTextDidEndEditing(_ searchBar: UISearchBar) {
         var snapshot = self.dataSource.snapshot()
-        let items: [PeopleCollectionViewDataSource.ItemType] = self.allPeople.map { person in
-            return .person(person)
+        let items: [PeopleCollectionViewDataSource.ItemType] = self.allPeople.sorted().map { person in
```

Comment:
Added a sort and custom selection logic as we reset the collection view but want to maintain what items are selected.